### ISSUE TITLE
Support for connecting to Electrum server over Tor

### ISSUE THREAD

**Issue by katoog** [SatoshiPortal/bullbitcoin-mobile#299]
Please add support to connect to custom Electrum server with Tor address.

**Comment by BullishNode** [comment]
concept ACK

**Comment by i5hi** [comment]
To start with we can quite easily support setting a proxy port and using an external Tor daemon like Orbot. 

Additionally, bdk supports Tor but lwk currently does not. Will request lwk to add this.

**Comment by thibistaken** [comment]
Yes, @emtreulapollaguera we'll prioritize this in our upcoming release in December.
